### DISCUSSION TITLE
Doubled dataset lenght

### DISCUSSION THREAD

**Discussion by copybaras** [d8ahazard/sd_dreambooth_extension#1052] [Q&A]
I have 17 images and 17 txt files with prompts for this images in dataset folder.
When starting training I have "Total dataset length (steps): 34" message. Is it fine or should I delete txt from the folder?

**Comment by minienglish1** [comment]
Txt isn't the problem.  Do you have classification images?  Maybe it generated 1 classification image per dataset image.

**Reply by copybaras** [reply]
I tried 5-8-10 classification images per dataset image in several different new models.
Anyway after restart of webui it's now proper 17. Don't know what caused the problem. 
Will pay more attention next time, maybe I reproduce it .
